### PR TITLE
Separators

### LINKED ISSUE #13: Needs special symbols support
For some services needs special symbols support. For example,

> Password must contain at least one special character.

### PR DESCRIPTION
**Pull request by omgovich**
Closes #13

New `separators` option. 

1) Useful for passwords that need special symbols (`separators: "$=_-@"`).
2) Useful to generate passphrases (`separators: " "`).

### CODE REVIEW

File: test/index.test.js [hunk 1]
```diff
@@ -61,3 +61,22 @@ test("change characters", () => {
...
+test("generate passphrase", () => {
+  const password = generatePassword({
+    minSyllableLength: 4,
+    maxSyllableLength: 6,
+    hasNumbers: false,
+    titlecased: false,
+    separators: " "
+  });
+  const regexp = /^[a-z]{4,6} [a-z]{4,6} [a-z]{4,6}$/;
+  console.log(password);
```
Comment (molefrog) [inline test/index.test.js]: ☝️

File: test/index.test.js [hunk 2]
```diff
@@ -61,3 +61,22 @@ test("change characters", () => {
   const regexp = /^([А-Я][а-я]{1,2}[0-9]){3}/;
   expect(regexp.test(password)).toBe(true);
 });
+
+test("enable separators", () => {
+  const password = generatePassword({ separators: "<>" });
+  const regexp = /^([A-Z][a-z]{1,2}[0-9][<>]){2}[A-Z][a-z]{1,2}[0-9]/;
```
Comment (molefrog) [inline test/index.test.js]: Just an idea. Regexp-based tests are a bit hard to maintain and read. As an alternative, I can suggest using a "property-based testing" approach. 

For example in this test case, we don't really need to know the exact shape of the resulted password. We really don't care here if the phrases divided by the separators contain digits or not. Currently, the coupling between those test cases is high: if you change the algorithm of how one word is generated this case will break as well. But, in reality, what we really care is this: "when separators option is specified it produces a sequence of words divided by those separators". So we could be asserting:
1. (okeish assertion) The resulted string should contain at least one of the separators.
2. (stronger) Split the resulted string using separators and count number of parts and the length of a word (min/max).

Comment (molefrog) [reply test/index.test.js]: https://medium.com/@vporoshok/property-based-testing-642cb7f9817f

Comment (omgovich) [reply test/index.test.js]: See 24da768 pls. Does it look better?

Comment (molefrog) [reply test/index.test.js]: Yes!

**Pull request merged**